pkg/linode-bs/deploy/releases/linode-blockstorage-csi-driver-v0.5.1.yaml

```diff
@@ -0,0 +1,563 @@
+          valueFrom:
+            secretKeyRef:
+              key: token
+              name: linode
+        image: linode/linode-blockstorage-csi-driver:v0.5.0
+        name: linode-csi-plugin
+        volumeMounts:
+        - mountPath: /linode-info
+          name: linode-info
+        - mountPath: /scripts
+          name: get-linode-id
+        - mountPath: /var/lib/csi/sockets/pluginproxy/
+          name: socket-dir
+      initContainers:
+      - command:
+        - /scripts/get-linode-id.sh
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        image: bitnami/kubectl:1.16.3-debian-10-r36
+        name: init
+        volumeMounts:
+        - mountPath: /linode-info
+          name: linode-info
+        - mountPath: /scripts
+          name: get-linode-id
+      serviceAccount: csi-controller-sa
+      tolerations:
+      - effect: NoSchedule
+        operator: Exists
+      - key: CriticalAddonsOnly
+        operator: Exists
+      - effect: NoExecute
+        operator: Exists
+      volumes:
+      - emptyDir: {}
+        name: socket-dir
+      - emptyDir: {}
+        name: linode-info
+      - configMap:
+          defaultMode: 493
+          name: get-linode-id
+        name: get-linode-id
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: csi-linode-node
+  name: csi-linode-node
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: csi-linode-node
+  template:
+    metadata:
+      labels:
+        app: csi-linode-node
+        role: csi-linode
+    spec:
+      containers:
+      - args:
+        - --v=2
+        - --csi-address=$(ADDRESS)
+        - --kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)
+        env:
+        - name: ADDRESS
+          value: /csi/csi.sock
+        - name: DRIVER_REG_SOCK_PATH
+          value: /var/lib/kubelet/plugins/linodebs.csi.linode.com/csi.sock
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v1.3.0
+        name: csi-node-driver-registrar
+        volumeMounts:
+        - mountPath: /csi
+          name: plugin-dir
+        - mountPath: /registration
+          name: registration-dir
+      - args:
+        - --endpoint=$(CSI_ENDPOINT)
+        - --token=$(LINODE_TOKEN)
+        - --url=$(LINODE_API_URL)
+        - --node=$(NODE_NAME)
+        - --v=2
+        env:
+        - name: CSI_ENDPOINT
+          value: unix:///csi/csi.sock
+        - name: LINODE_API_URL
+          value: https://api.linode.com/v4
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: LINODE_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: token
+              name: linode
+        image: linode/linode-blockstorage-csi-driver:v0.5.0
```

Comment:
Should this image also updated to v0.5.1? I see that in other versioned yaml's, the version is kept to the version of yaml file.